app/src/main/java/com/github/libretube/ui/sheets/IntentChooserSheet.kt

```diff
@@ -26,6 +26,7 @@ class IntentChooserSheet(
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        val binding = binding
```

Comment:
What exactly is this doing?

Reply:
It's calling the non-null getter method and holding the value.

Reply:
Interesting, never seen that before.

Reply:
But makes sense, yeah.